src/instruments/src/DCDU/index.tsx

```diff
@@ -56,29 +59,44 @@ const DCDU: React.FC = () => {
         if (sender.length === 0 || sender === statusMessage.sender) {
             state.sender = '';
             state.message = '';
-            state.remainingMilliseconds = 0;
             setStatusMessage(state);
+            if (messageStatusTimeout) {
+                clearTimeout(messageStatusTimeout);
+                setMessageStatusTimeout(null);
+            }
         }
     };
-    const setStatus = (sender: string, message: string) => {
+    const setStatus = (sender: string, message: string, duration: number) => {
         const state = statusMessage;
         state.sender = sender;
         state.message = message;
-        state.remainingMilliseconds = 5000;
         setStatusMessage(state);
+        if (duration > 0) {
```

Comment:
@svengcz How about instead of using the magic -1 you use `duration: Infinity` and check `if (duration.isFinite())` here? (because magic numbers are evil :-)

Reply:
fixed